cmake/scripts/expand_instantiations.cc

```diff
@@ -439,7 +439,18 @@ process_instantiations()
   //   for (X:Y; A:B) { INST }
   while (whole_file.size() != 0)
     {
+      // skip space, tabs, comments:
       skip_space(whole_file);
+
+      // output preprocessor defines as is:
+      if (has_prefix(whole_file, "#"))
+	{
```

Comment:
The indentation looks weird. Can you replace the tabs?

Reply:
done.